src/app/dashboard/overview/components/TransferHistory.jsx

```diff
@@ -1,62 +1,87 @@
-                        <Table.Cell className='max-w-fit'>
-                          <HiDownload size={20} />
-                        </Table.Cell>}
-                      {asset.transfer_out &&
-                        <Table.Cell className='max-w-fit'>
-                          <HiUpload size={20} />
-                        </Table.Cell>}
-                      {!asset.transfer_out && !asset.transfer_in && asset.status === 'Pending' &&
-                        <Table.Cell className='max-w-fit'>
-                          <HiDotsHorizontal size={20} />
-                        </Table.Cell>}
-                      {!asset.transfer_out && !asset.transfer_in && asset.status === 'Failed' &&
-                        <Table.Cell className='max-w-fit'>
-                          <HiExclamationCircle size={20} />
-                        </Table.Cell>}
-                      <Table.Cell className='whitespace-nowrap font-normal text-black dark:text-white flex flex-row gap-2 ml-0'>
-                        {asset.asset}
-                      </Table.Cell>
-                      <Table.Cell className='whitespace-nowrap font-normal text-black dark:text-white'>{asset.counterpart_id}</Table.Cell>
-                      <Table.Cell className='whitespace-nowrap font-normal text-black dark:text-white'>{asset.status}</Table.Cell>
-                      <Table.Cell className='whitespace-nowrap font-normal text-black dark:text-white'>{asset.date}</Table.Cell>
-                    </Table.Row>
-                  )
-                })}
-              </Table.Body>
-            </Table>
+    <>
+      {!transferHistory && ( // Error during fetch (potentially Connector is unvailable)
+        <div className='flex flex-col w-full gap-4 p-4'>
+          <div className='w-full gap-4 p-6 text-center bg-white border border-gray-200 rounded-lg shadow'>
+            <h5 className='mb-2 text-2xl font-bold tracking-tight text-gray-900'>Connector service unavailable. Please try again later.</h5>
           </div>
-        </Accordion.Content>
-      </Accordion.Panel>
-    </Accordion>
+        </div>
+      )}
+      {transferHistory?.error && (
+        <div className='flex flex-col w-full gap-4 p-4 bg-gray-50'>
+          <Alert color='failure' icon={TbAlertSquareFilled}>
+            <span className='font-bold text-xl'>Connector service is not responding. Please, try again.</span>
+          </Alert>
+        </div>
+      )}
+      <Accordion className='mt-5 mb-5 p-10 pt-0 border-none'>
+        <AccordionPanel>
+          <AccordionTitle>
+            <div className='flex flex-row gap-2'>
+              <svg className='w-6 h-6 text-gray-800 dark:text-white' aria-hidden='true' xmlns='http://www.w3.org/2000/svg' width='24' height='24' fill='none' viewBox='0 0 24 24'>
+                <path stroke='currentColor' strokeLinecap='round' strokeLinejoin='round' strokeWidth='2' d='M12 8v4l3 3m6-3a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z' />
+              </svg>Transfer History
```

Comment:
Suggestion: since users have no control on this request until we implement #89 , and it is sorted by date, we could reword this to:
```suggestion
              </svg>Recent Transfers
```